.gitignore

```diff
@@ -1,3 +1,7 @@
 *.swp
 *~
 .DS_Store
+*.iml
+*.idea
```

Comment:
This is true. But beyond .gitignore, I've yet to see this cause an issue in any of our other repos. Otherwise, you keep adding random things ad nauseam.